src/test/java/uk/gov/digital/ho/pttg/application/HmrcHateoasClientTest.java

```diff
@@ -392,14 +393,14 @@ public void shouldLogAfterEmploymentResourceResponseReceived() {
     public void shouldLogBeforeGetSelfAssessmentResourceRequestSent() {
         given(mockHmrcCallWrapper.exchange(any(URI.class), eq(HttpMethod.GET), any(HttpEntity.class), any(ParameterizedTypeReference.class))).willReturn(mockResponse);
 
-        client.getSelfAssessmentResource("token", LocalDate.of(2000, 1, 1), LocalDate.of(2000, 2, 2), new Link("http://something.com/anyurl"));
+        client.getSelfAssessmentResource("token", "2010-11", "2011-12", new Link("http://something.com/anyurl"));
 
         then(mockAppender)
                 .should()
                 .doAppend(argThat(argument -> {
                     LoggingEvent loggingEvent = (LoggingEvent) argument;
 
-                    return loggingEvent.getFormattedMessage().equals("About to get self assessment resource from HMRC at http://something.com/anyurl?fromTaxYear=1999-0&toTaxYear=1999-0") &&
```

Comment:
Was there any significance in this tax year being malformed?

Reply:
I'm quite confident that there isn't. Any date before 2010 will produce the malformed date because of how the getTaxYear method works. This test was added in with a lot of tests to check that we do logging as part of EE-13694, which was to add durations to the logs.

As for the malformed dates - we don't retrieve data from before 2010 and the bug won't hit us until the year 2100 I believe. But I may fix it to be on the safe side (I'm already in this bit of the code).